blog/01-22-23-dynamic-nft-creation-using-svg-to-build-in-smart-contract-on-celo/dynamic-nft-creation-using-svg-to-build-in-smart-contract-on-celo.md

```diff
@@ -0,0 +1,1038 @@
+---
+title: Dynamic NFT Creation using SVG to build in Smart Contract on Celo.
+description: Welcome to our tutorial on dynamic NFT creation using SVG to build in a smart contract on Celo!
+authors:
+  - name: Kunal Dawar
+url: https://github.com/developerkunal
+image_url: https://avatars.githubusercontent.com/u/35455566?v=4
+tags: ["intermediate", "solidity", "celosage", "erc721", "truffle"]
+hide_table_of_contents: false
+slug: /tutorials/dynamic-nft-creation-using-svg-to-build-in-smart-contract-on-celo
+---
+

```

Comment:
Add a header image just after frontmatter. The image should look something like this - `![header](../src/data-tutorials/showcase/<beginner/intermediate/advance>/<image-name>.png)`

Reply:
@viral-sangani can you help me out a bit my laptop got issues its not turning on if its possible can you add the header image for this time only. I will make sure next time to insert it.

Reply:
@viral-sangani i have fixed the header.